contracts/DADBridgeWrapper.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.9;
+
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { Pausable } from "@openzeppelin/contracts/security/Pausable.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+error InvalidToken(IERC20 token);
+error InvalidRecipient(address recipient);
+error InvalidAmount(uint256 amount);
+error UnsupportedOperation();
+
+contract DADBridgeWrapper is AccessControl, Pausable {
+    using SafeERC20 for IERC20;
+
+    bytes32 public constant ROLE_ADMIN = keccak256("ROLE_ADMIN");
+
+    IERC20 private immutable _token;
+
+    uint256 private _totalSupply;
+
+    /**
+     * @dev initializes the contract
+     */
+    constructor(IERC20 tokenAddr) {
```

Comment:
can just call it `token`, that's what we do everywhere

Reply:
Or `initToken`, when a `token` getter already exists

Reply:
Fixed.